Jessy/Nba_Challenge.py

```diff
@@ -0,0 +1,101 @@
+"The Lakers ended the season atop the Western Conference with a record of 49-14. They were narrowly behind the Bucks for the best record in \n" 
+"the league. Davis proved to the final piece necessary for the Lakers to rebound from missing the playoffs last year. Los Angeles was a \n"
+"dominant club on both sides of the ball and are in a position to have another successful year next season."
+print('Challenge 3.2: Print out the paragraph but with only 1 sentence per line')
+# TODO: As above, orint out the paragraph with only 1 sentence per line, and all in upper case
+print("The Lakers went all in this offseason and swung a deal for former Pelicans forward Anthony Davis.\n They sent a package of Brandon Ingram,"
+ "Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis.\n Those three have made good developments with the Pelicans, "
+  "especially Brandon Ingram.\n But, the deal is still a huge win for the Lakers as Lebron, Davis, and company have put together an incredible "
+  "season.\n Los Angeles has ridden James and Davis, along with a supporting cast built around them, to the second-best record in the NBA. \n" 
+  "The Lakers ended the season atop the Western Conference with a record of 49-14.\n They were narrowly behind the Bucks for the best record in " 
+  "the league.\n Davis proved to the final piece necessary for the Lakers to rebound from missing the playoffs last year.\n Los Angeles was a "
+  "dominant club on both sides of the ball and are in a position to have another successful year next season.".upper())
+
+print('Challenge 3.3: Make a boolean variable indicating whether you think the Lakers are the best NBA team')
+# TODO: make a variable called `lakers_are_best` to indicate this
+lakers_are_best= True
+# TODO: print out the variable in an f-string to convey your opinion on the lakers
+print(f"Lakers are the best {lakers_are_best}")
+
+print('Challenge 3.4: Type Conversion')
+# TODO: Convert your `lakers_are_best` variable to an integer, and print it out. 
+My_int_lakers= int(lakers_are_best)
+print(My_int_lakers)
+# TODO: Convert your `lakers_are best` variable to a float, and print it out
+My_float_lakers= float(lakers_are_best)
+print(My_float_lakers)
+print('Challenge 3.5: Type Conversion Part 2')
+# TODO: Take each player's three point percentage (from part 2.5) and convert it to a string, then print it out.
+My_string_Jamalper= str(Jamal_Percentage)
+My_string_FredPer= str(Fred_Percentage)
+My_string_JamesPer= str(James_Percentage)
+print(My_string_Jamalper)
+print(My_string_FredPer)
+print(My_string_JamesPer)
+
+# TODO: Take each player's three point percentage (from part 2.5) and convert it to an integer, then print it out.
+My_int_Jamalper= int(Jamal_Percentage)
+My_int_Fredper= int(Fred_Percentage)
+My_int_Jamesper= int(James_Percentage)
+print(My_int_Jamalper)
+print(My_int_Fredper)
+print(My_int_Jamesper)
```

Comment:
Great work - but if you use the function int( ) on a decimal like 0.49, when you run int(0.49) the outcome is 0, because python removes everything after the decimal

what you need to do, multiply the percents by 100                  so 0.49*100 = 49.0

you can also get fancy and add "%" , how could we?